source/MaterialXRender/ShaderValidators/Osl/OslValidator.cpp

```diff
@@ -169,7 +169,7 @@ void OslValidator::renderOSL(const std::string& outputPath, const std::string& s
     std::string command(_oslTestRenderExecutable);
     command += " " + sceneFileName;
     command += " " + outputFileName;
-    command += " --path " + osoPaths;
+    command += " -r 512 512 --path " + osoPaths;
```

Comment:
Should there be a parameter for the image size? Doesn't have to be in this review, just curious.